py-scoring/src/ai/h2o/sparkling/ml/params/H2OMOJOModelParams.py

```diff
@@ -46,6 +46,12 @@ def getTrainingParams(self):
     def getModelCategory(self):
         return self._java_obj.getModelCategory()
 
+    def getScoringHistory(self):
```

Comment:
is it documented anywhere what should users expect on the output?

Reply:
Currently, It's not documented. I will improve that.